pipeline/patch_VAE.py

```diff
@@ -392,13 +384,16 @@ def process_VAE(raw_folder: str,
     assert len(channels) > 0, "At least one channel must be specified"
 
     # these sites should be from a single condition (C5, C4, B-wells, etc..)
-    # model_path = os.path.join(weights_dir, 'model.pt')
-    # model_name = os.path.basename(weights_dir)
-    # output_dir = os.path.join(raw_folder, model_name)
-    # os.makedirs(output_dir, exist_ok=True)
-    # output_dir = config_.inference.weights
-    output_dir = raw_folder
+    model_dir = os.path.dirname(model_path)
+    model_name = os.path.basename(model_dir)
```

Comment:
We could add `model_name` to the config

Reply:
Sure. `model_name` here is only used to name the output folder so it can be user specified. We can make the default same as the name of the model folder (which is parsed from `model path`)